src/cljs/rems/table.cljs

```diff
@@ -18,16 +18,21 @@
 (defn column-class [column-definitions col]
   (get-in column-definitions [col :class] (name col)))
 
+(defn column-class-customized [item column-definitions col]
+  (if (= "draft" (:state item))
+    "text-warning"
```

Comment:
Warning style is risky to use here because an existing draft is generally not something to warn about but to advertise in a positive sense. Maybe introduce a separate `text-highlight` style?

Reply:
👍